_episodes/51.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: episode
+title: "51: Trino cools off with PopSQL"
+date: 2023-07-06
+tags: trino popsql query editor collaboration
+youtube_id: ""
+wistia_id: ""
+sections:
+- title: "Intro"
+  desc: "An intro to PopSQL"
+  time: 0
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* Jake Peterson, Head of Customer Success at [PopSQL](https://popsql.com/)
```

Comment:
Need to ask if they want other links

Reply:
@mosabua @colebow you can include my linkedin too if you'd like, but just linking to PopSQL is fine too https://www.linkedin.com/in/jakeptrsn/